Tests/Marketplace/zip_packs.py

```diff
@@ -34,6 +33,8 @@ def option_handler():
     parser.add_argument('-b', '--bucket_name', help="Storage bucket name", required=True)
     parser.add_argument('-br', '--branch_name', help="Name of the branch", required=False)
     parser.add_argument('-n', '--circle_build', help="Number of the circle build", required=False)
```

Comment:
You've removed the use of a couple of the script's arguments in the `main` function but didn't remove them from the `options_handler`. Please ensure these are synced.

Reply:
Fixed.